.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - macos-latest
+          - ubuntu-latest
+        target:
+          - 0.20.2
+          - 1.0.10
+          - 1.2.8
+          - 1.4.0
+          - git:8a004e2
+          - nightly:https://github.com/nim-lang/nightlies/releases/tag/2020-11-11-devel-bbe49a14ae827b6474d692042406716a3b3dd71f
```

Comment:
I'm guessing choosenim is not used for this case?

Reply:
Correct. If there's a `something:` prefix, choosenim is not used.